modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/test/java/com/liferay/portal/search/elasticsearch7/internal/query/ElasticsearchQueryTranslatorTest.java

```diff
@@ -115,6 +145,22 @@ private void _assertBoost(Query query) {
 			String.valueOf(queryBuilder.boost()));
 	}
 
+	private void _assertNumberOfTerms(int expected, TermsQuery termsQuery) {
```

Comment:
maybe rename this to _assertNumberOfTermsClauses so its a little more clear whats happening